libs/vkd3d/command.c

```diff
@@ -2541,6 +2577,12 @@ static void d3d12_command_list_emit_render_pass_transition(struct d3d12_command_
                 mode, list->dsv_layout, do_clear, &vk_image_barriers[j++]);
     }
 
+    if (list->vrs_image)
+    {
+        stage_mask |= vk_render_pass_barrier_vrs(list->vrs_image->vrs_view,
```

Comment:
Is it necessary to also pass down vrs_view when we pass down the d3d12_resource?

Reply:
I guess not